README.md

```diff
@@ -1,28 +1,153 @@
-# discovery
+# Discovery Service
 
-## Build
+## Main principles and features described
+### Features
+* Fetch MYST prices from the public API
+* Create prices for nodes based on the load and multiplier in the countries
+* Listen for proposals from NATS Broker and enhance them with quality metrics and tags
+* Run proposal expiration job
+* Update price config by Universe Admin
+* Using Postgres for saving proposals, configs, and country multipliers
```

Comment:
I think that the sentence could be phrased differently to be more consistent with the previous one.

Reply:
```suggestion
* Use Postgres to save proposals, configs, and country multipliers
```